test/locators.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { browser, ExpectedConditions } from 'protractor';
+import { PersonalInformationPage } from '../src/page';
+
+describe('Test of filling the form. Entering to the website', () => {
+  beforeAll(async () => {
+    await browser.get('https://www.tutorialspoint.com/selenium/selenium_automation_practice.htm');
+  });
+  describe('filling the form', async () => {
+    const personalInformationPage: PersonalInformationPage = new PersonalInformationPage();
+    beforeAll(async () => {
+      await personalInformationPage.fillForm({
+        firstName: 'Alejandro',
+        lastName: 'Perdomo',
+        sex: 'Male',
+        experience: 7,
+        profession: ['Automation Tester'],
+        tools: ['Selenium Webdriver'],
+        continent: 'South America',
+        commands: [
+          'Browser Commands',
+          'Navigation Commands',
+          'Switch Commands',
+          'Wait Commands',
+          'WebElement Commands']
+      });
+      await personalInformationPage.finishForm();
+    });
+    it('The submit button should has been pressed', async () => {
+      await expect(ExpectedConditions.alertIsPresent());
```

Comment:
The idea here is to make an assertion to the "Selenium - Automation Practice Form" title, after the accept button of the alert is pressed. Probably you will have to create another function on the `personalInformationPage` PageObject to click on the accept on the alert button. You will need to investigate how to interact with alerts 😉 .

Reply:
I needed to use the capability "unexpectedAlertBehaviour: 'accept'", because Protractor or Node, I am not sure, was handling this unexpected alert like an unhandled promise error, so Protractor was closing the alert automatically and then I could not interact with the alert. 

I know how to interact with alerts https://stackoverflow.com/questions/41468821/how-to-handle-unexpectedalertopenerror-in-protractor

Reply:
what if you use the `switchTo().alert()`?

Reply:
Yes, I used it, but Node before I can interact with the alert, it closes the alert, because it takes the alert like an unhandled promise. I saw the swichTo().alert() in the StackOverflow post that I mentioned

Reply:
I cloned your code, it seems that there are some async awaits missing because the alert box appears before the form ends. Maybe I'm wrong, I'm investigating what else could be the cause of the error.

Reply:
@StevenMaMei I was debugging, it seems that the button is clicked first and then the commands are selected.

Reply:
> @StevenMaMei I was debugging, it seems that the button is clicked first and then the commands are selected.

jumm, that doesn't make sense, I have all the sync/await. I wil check my code.

Reply:
@StevenMaMei  I had to add some async awaits, and call to my uncle juaquin hehe.  Please change the order in what options are filled, please put the commands before tools and continent. Then handle the alert 😄

Reply:
ok!

Reply:
The test now works good. I did what you say, yet I had to do something additional. I had to put:
unexpectedAlertBehaviour: 'ignore' in the config files.
Otherwise an error will occur. That config is to tell to Node that unexpected alerts are not an error.
![image](https://user-images.githubusercontent.com/33093117/88993800-87a27400-d2ac-11ea-8b38-f4b6fe0d36ad.png)

Reply:
I didn't need anything additional to change the order in what the options are filled and put the alert handling on the finishForm method after the click on the button.
Please update this branch in order to help you :)

Reply:
> I didn't need anything additional to change the order in what the options are filled and put the alert handling on the finishForm method after the click on the button.
> Please update this branch in order to help you :)

Done

Reply:
Nevermind, the test are working without that config. Thanks for you help =).